### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by HarveyPlante** [HarveyPlante/codelab-multipage-websites#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](http://eminent-beam.surge.sh) and meets the following [project](http://lansingcodelab.com/lessons/multipage-websites/1) criteria:
- [x] a `/students` route exists, that lists the first names of 3 other Code Lab students (seek their permission first - if they're not comfortable with their first name on the page, you can make up a name)
- [x] a route for each student exists at `/students/NAME`, where their name and favorite color is featured

**Comment by chrisvfritz** [comment]
Looks fantastic! 🎉  All of the criteria are definitely met, but since there's nothing else to improve, I'm going to help you get rid of those `.DS_Store` files [cluttering up the project](https://github.com/HarveyPlante/codelab-multipage-websites). Check out [this question on StackOverflow](http://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository#answer-107921). The first answer is what I would use to solve the problem. Note that all the commands they list are terminal commands. 😃

**Comment by chrisvfritz** [comment]
Let me know here when you've updated the code on GitHub!

**Comment by HarveyPlante** [comment]
Okay so I figured out how to get rid of the annoying .DS_Store file -- Wow
I learned  a lot by doing this..
--figured out how to add a file in the terminal then how to edit with NANO
text editor--
This article also helped me https://help.github.com/articles/ignoring-files/

--also learned some more commands to use in Git and have become more
comfortable using it--

I may have messed up the codelab-js-intro in Github though ( I deleted
.DS_Store file on the site manually)..I cannot push new code to it.

Thanks Chris! I'll see you tonight.
